yt_dlp/extractor/minitv.py

```diff
@@ -0,0 +1,337 @@
+      images
+      description {
+        synopsis
+        contentLengthInSeconds
+      }
+      publicReleaseDateUTC
+      audioTracks
+      seasonId
+      seriesId
+      seriesName
+      seasonNumber
+      episodeNumber
+      timecode {
+        endCreditsTime
+      }
+    }
+    ... on MovieContent {
+      contentId
+      vodType
+      name
+      description {
+        synopsis
+        contentLengthInSeconds
+      }
+      images
+      publicReleaseDateUTC
+      audioTracks
+    }
+  }
+}''',
+            },
+            note='Downloading title info')
+
+        prs = self._call_api(asin, note='Downloading playback info')
+
+        formats = []
+        for type_, asset in prs['playbackAssets'].items():
+            if not isinstance(asset, dict):
+                continue
+            if type_ == 'hls':
+                formats.extend(self._extract_m3u8_formats(
+                    asset['manifestUrl'], asin, ext='mp4', entry_protocol='m3u8_native',
+                    m3u8_id=type_, fatal=False))
+            elif type_ == 'dash':
+                formats.extend(self._extract_mpd_formats(
+                    asset['manifestUrl'], asin, mpd_id=type_, fatal=False))
+            else:
+                pass
+        self._sort_formats(formats)
+
+        duration = traverse_obj(title_info, ('description', 'contentLengthInSeconds'))
+        chapters = []
+        credits_time = try_get(title_info, lambda x: x['timecode']['endCreditsTime'] / 1000)
+        if credits_time is not None and duration is not None:
+            chapters.append({
+                'start_time': credits_time,
+                'end_time': duration,
+                'title': 'End Credits',
+            })
+
+        info = {
+            'id': asin,
+            'title': title_info['name'],
+            'formats': formats,
+            'language': traverse_obj(title_info, ('audioTracks', 0)),
+            'thumbnails': [{
+                'id': type_,
+                'url': url,
+            } for type_, url in title_info.get('images', {}).items()],
+            'description': traverse_obj(title_info, ('description', 'synopsis')),
+            'release_timestamp': int_or_none(try_get(title_info, lambda x: x['publicReleaseDateUTC'] / 1000)),
+            'duration': duration,
+            'chapters': chapters,
+            'series': title_info.get('seriesName'),
+            'series_id': title_info.get('seriesId'),
+            'season_number': title_info.get('seasonNumber'),
+            'season_id': title_info.get('seasonId'),
+            'episode_number': title_info.get('episodeNumber'),
+        }
+        if title_info.get('vodType') == 'EPISODE':
+            info.update({
+                'episode': title_info['name'],
+                'episode_id': asin,
+            })
+
+        return info
+
+
+class MiniTVSeasonIE(MiniTVIE):
+    IE_NAME = 'minitv:season'
+    _VALID_URL = r'minitv:season:(?:amzn1\.dv\.gti\.)?(?P<id>[a-f0-9-]+)'
```

Comment:
this doesnt have URL?

Reply:
I could add it just in case, but the URLs you get from the website always seem to point to a single episode.

Reply:
Write an ie_desc similar to this so users know what prefix to use. (by looking at supportedsites)

```
IE_DESC = 'Amazon MiniTV Series, "minitv:season:" prefix'
```